wpilibNewCommands/src/test/java/edu/wpi/first/wpilibj2/command/CommandDecoratorTest.java

```diff
@@ -44,31 +41,37 @@ void withTimeoutTest() {
   @Test
   void untilTest() {
     try (CommandScheduler scheduler = new CommandScheduler()) {
-      AtomicBoolean condition = new AtomicBoolean();
+      AtomicBoolean finish = new AtomicBoolean();
 
-      Command command = new WaitCommand(10).until(condition::get);
+      Command command = new RunCommand(() -> {}).until(finish::get);
```

Comment:
All these instances of `RunCommand(() -> {})` can be replaced with `Commands.idle()`.

Not important enough to block this PR, though.